spec/controllers/service_uploads_controller_spec.rb

```diff
@@ -152,7 +153,7 @@ def make_request
         let(:educator) { FactoryGirl.create(:educator) }
         it 'cannot access the page; gets redirected' do
           make_request
-          expect(JSON.parse(response.body)).to eq({ "error" => "You don't have the correct authorization." })
+          expect(response).to redirect_to(not_authorized_path)
```

Comment:
This endpoint returns HTML, so it should have been redirecting, not returning JSON.